nextflow.config

```diff
@@ -9,19 +9,21 @@ docker {
 }
 
 params {
-    inputDir=''
-    outputDir=''
-    localOutDir=''
-    outPrefix='test.'
-    cpus=1
-    limitMemory=32000000
+    genome='/path/to/genome'
+    inputDir='path/to/input/folder'
+    inputPattern="*{1,2}.fastq.gz"
+    outputDir='/path/to/output/folder'
+    starTmpDir='/path/to/STAR/temp/folder'
+    localOutDir='.'
```

Comment:
Since you named 'outputDir' earlier, I'd name this 'localOutputDir'. Super-minor... ignore it :)